demos/projects/ST/b-l475e-iot01a/ADU.md

```diff
@@ -30,6 +30,15 @@ This sample will allow you to update an STMicroelectronics B-L475E-IOT01A Discov
 1. [GNU Arm Embedded Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) (Version 9 or higher)
 1. Serial terminal tool like [Termite](https://www.compuphase.com/software_termite.htm), Putty, Tera Term, etc.
 
+To run this sample you can use a device previously created on your Azure IoT Hub or you may have the Azure IoT middleware for FreeRTOS provision your device automatically using Azure Device Provisioning Services (DPS).
+
+IoT Hub | DPS
+---------|----------
+Have an [Azure IoT Hub](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal) created | Have an instance of [IoT Hub Device Provisioning Service](https://docs.microsoft.com/azure/iot-dps/quick-setup-auto-provision#create-a-new-iot-hub-device-provisioning-service)
+Have a [logical device](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal#register-a-new-device-in-the-iot-hub) created in your Azure IoT Hub using your preferred authentication method* | Have an [individual enrollment](https://docs.microsoft.com/azure/iot-dps/how-to-manage-enrollments#create-a-device-enrollment) created in your instance of DPS using your preferred authentication method*
+
+**While this sample supports SAS keys and Certificates, this guide will refer only to SAS keys.**
```

Comment:
Did you mean to have a star here which correlates to the cells above for auth (not sure if markdown needs it escaped)?
```suggestion
**\*While this sample supports SAS keys and Certificates, this guide will refer only to SAS keys.**
```

Reply:
Ooh, good catch! I'll update this in the main readme as well

Reply:
I updated this to not have the bold styling since all of the readme's that had the asterisk properly showing just had the asterisk